src/components/v2/V2Project/V2FundingCycleSection/FundingCycleDetails.tsx

```diff
@@ -295,6 +295,25 @@ export default function FundingCycleDetails({
             )}
           </FundingCycleDetailWarning>
         </Descriptions.Item>
+        <Descriptions.Item
```

Comment:
Adds this to the FC card on project page.

<img width="530" alt="Screen Shot 2022-07-05 at 10 51 58 AM" src="https://user-images.githubusercontent.com/94939382/177231546-7dd6c953-70ae-4085-89fa-316a6a1b1d39.png">